app/src/main/java/com/example/coinage/fragments/GoogleSheetsFragment.java

```diff
@@ -254,11 +252,17 @@ protected Void doInBackground(Void... voids) {
             } catch (JSONException e) {
                 e.printStackTrace();
             }
-            return null;
+            return true;
         }
 
-        protected void onPostExecute() {
-            Log.i(TAG, "Spreadsheet created successfully!");
+        @Override
+        protected void onPostExecute(Boolean doInBackgroundResult) {
+            if (doInBackgroundResult) {
```

Comment:
nit - this is super nit picky but an interesting discussion -

in the context of the function `onPostExecute`, should this variable be called `doInBackgroundResult`? Within this function, the way with which we fetched the variable (`inBackground`) is "abstracted away" from the function. This just means we should name the variable based on *what it means in the method*, something like "isSuccess".